docs/ssh-keys/test-ssh-connection.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Test SSH connection
```

Comment:
We should guide our users beyond only testing connections or SSH keys. Our users need to effectively connect to their servers using SSH and use their servers the way they want.

I believe we actually need a topic `servers/ssh-into-server` and, if needed, on `ssh-keys/add-ssh-keys` we could link it as a next step.

What do you think?